### DISCUSSION TITLE
ネザー 禍壺の砦B 開発状況

### DISCUSSION THREAD

**Discussion by anminmakura** [TUSB/TheUnusualSkyBlock#75] [エリア・建築の完成度・開発状況]
**壺内部の砦について**
座標地点 ネザー 227 86 862
- [ ] 土台の製作
- [ ] 建築に着手
- [ ] 装飾に着手
- 要望
某クッパ城をイメージして砦を作ってほしい。
特にここは大規模なので１人でやるのはきついと思う。

**Comment by anminmakura** [comment]
![227_86_862](https://user-images.githubusercontent.com/39670701/96727450-03b0e180-13ee-11eb-9f7c-17b36f10b38c.png)

**Comment by reiOmoti** [comment]
![2021-01-01_15 22 45](https://user-images.githubusercontent.com/76546701/103434504-8f908680-4c45-11eb-8402-fae3bac98d4c.png)
こんなの如何でしょう...?

**Comment by anminmakura** [comment]
氷塊ブロックを使っていること以外はかなり良いと思います。
特に土台の模様が丁寧です。

ネザーに氷はアクセントとしては良いですが、広範囲に使うと違和感があるので一か所に集中するぐらいの使い方であればいいとは思う。

**Comment by reiOmoti** [comment]
> 氷塊ブロックを使っていること以外はかなり良いと思います。
> 特に土台の模様が丁寧です。
> 
> ネザーに氷はアクセントとしては良いですが、広範囲に使うと違和感があるので一か所に集中するぐらいの使い方であればいいとは思う。

了解です～後ほど修正してschemで投げます、

**Comment by reiOmoti** [comment]
↑のschem(外装のみ)
内装や、細かい装飾は鯖内で...。(案募集したい感じ)
[rTU.zip](https://github.com/TUSB/TheUnusualSkyBlock/files/5958967/rTU.zip)

**Comment by anminmakura** [comment]
Bはまだ全然できていない。
過去にれい氏が進めていたので後で進捗を聞いてみる。

**Comment by reiOmoti** [comment]
製作再開:=>内装は6割完成。
　　　　 =>今後は内装の続き、及び土台内製作。恐らく1週間もあれば大まかな形は出来る予定。装飾類は鯖内に入れてから。
![2021-06-21_00 03 11](https://user-images.githubusercontent.com/76546701/122679145-b9099280-d224-11eb-9c81-c16de99a0978.png)
![2021-06-21_00 03 23](https://user-images.githubusercontent.com/76546701/122679146-bad35600-d224-11eb-8ccb-b5bdfc079977.png)
![2021-06-21_00 03 33](https://user-images.githubusercontent.com/76546701/122679149-bc9d1980-d224-11eb-8692-3b9ef165cc80.png)

**Comment by anminmakura** [comment]
@reiOmoti その後の進捗どうですか( . _ .   ; )
